var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -61,6 +62,18 @@ class Openblas(MakefilePackage):
         multi=False
     )
 
+    variant(
+        'virtual_machine',
+        default=False,
+        description="Adding options to build openblas on Linux virtual machine"
+    )
+
+    variant(
+        'num_threads',
```

Comment:
maybe it's better to control this via environment variables, i.e.
```
export OPENBLAS_NUM_THREADS=4
```
?

p.s. Most blas/lapack in spack have `threads` variant to control which multi-threading to use (if use anything at all).

Reply:
Okay I can take it out. In my handcrafted system, I was defining it at compile time.